.github/workflows/cd.yml

```
@@ -22,26 +22,23 @@ jobs:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Verify package
+        run: cargo verify-project --manifest-path zefiro-core/zefiro-cwl-parser/Cargo.toml
+      - name: Set the release version
+        run: echo "RELEASE_VERSION=${GITHUB_REF:11}" >> "$GITHUB_ENV"
+      - name: Install Rust toolchain
+        uses: dtolnay/rust-toolchain@stable
+      # - name: Publish the zefiro-core library
+      #   run: |
+      #     cargo publish --manifest-path zefiro-core/Cargo.toml \
+      #       --locked --token ${{ secrets.CARGO_REGISTRY_TOKEN }}
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)